docs/features.rst

```diff
@@ -138,7 +136,7 @@ Your project is already an initialised Git repository and setuptools_ uses the
 information of tags to infer the version of your project with the help of
 `setuptools_scm`_.  To use this feature you need to tag with the format
 ``MAJOR.MINOR[.PATCH]`` , e.g. ``0.0.1`` or ``0.1``.
-Run ``python setup.py --version`` to retrieve the current `PEP 440`_-compliant version.
+.. Run ``python -m setuptools_scm`` to retrieve the current `PEP 440`_-compliant version.
```

Comment:
This is not officially supported yet. I recently submitted a PR for that.

Reply:
Nice! Didn't know that :-) I would love to see this happen. In general, we would come to a point where our docs no longer mention ``python setup.py ...`` while still providing the same functionality.